server/test/logout.spec.js

```diff
@@ -0,0 +1,21 @@
+const chai = require("chai");
+const chaiHttp = require("chai-http");
+const app = require("../app.js");
+
+chai.should();
+chai.use(chaiHttp);
+
+describe("Pinging the Logout controller: GET /logout", () => {
+  it("it should return 200 and logout response", (done) => {
+    chai
+      .request(app.server)
+      .get(`/auth/logout`)
```

Comment:
Just so we are aware, GET isn't the verb that makes the most sense for a /logout path! (see this [article](https://www.baeldung.com/logout-get-vs-post) for more details on why) For now though I think this is fine to leave as-is 👍🏼

The existing implementation may have thrown you off but this is the pitfall of developers building tests - we have to be careful not to be biased by our knowledge of the implementation and focus on specs!

Reply:
I see. May I change the Logout Route from GET to POST?

Reply:
Yes, please do so if possible :)

Reply:
Done